panther_analysis_tool/main.py

```diff
@@ -380,6 +382,7 @@ def upload_zip(
                 )
                 if not backend.feature_flags(flags_params).data.flags[0].treatment:
```

Comment:
nit unrelated to these changes: without expanding the code above, it wasn't quite clear what this line of code was doing... particularly what `flags_params` meant.

it might be more clear if we renamed `flags_params` to `correlation_rule_flags`... or better yet, we could create a helper function that wrapped this logic to make the whole thing easier to understand and also re-use.

e.g.
```
def is_feature_enabled(flag_name: str) -> bool:
    flags_params = FeatureFlagsParams(
        flags=[FeatureFlagWithDefault(flag=flag_name)]
    )
    return backend.feature_flags(flags_params).data.flags[0].treatment
```

and then:
```
if not is_feature_enabled(ENABLE_CORRELATION_RULES_FLAG):
    del resp_dict["correlation_rules"]
    del resp_dict["signals"]
```

Reply:
I like that. I will switch it